tck/server.py

```
@@ -0,0 +1,71 @@
+    # If the response is already an error response, return it directly
+    if isinstance(response, dict) and 'error' in response:
+        return response
+
+    # Build and return the success response
+    return build_json_rpc_success_response(response, request_id)
+
+def start_server(config: ServerConfig | None = None):
+    """Start the JSON-RPC server using Flask."""
+    if config is None:
+        config = ServerConfig()
+    logger.info(f"Starting TCK server on {config.host}:{config.port}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)